thermal_processing.py

```diff
@@ -70,7 +70,7 @@ def classify(recording, api, s3):
         logging.info("tag: %s (%.2f)", tag, tags[tag]["confidence"])
         api.tag_recording(recording, tag, tags[tag])
 
-    upload_tracks(api, recording, tagged_tracks)
+    upload_tracks(api, recording, classify_info["algorithm"], tagged_tracks)
```

Comment:
I don't think I've seen the classifier-pipeline change which causes it to return the algorithm yet. I presume that's coming?